src/main/java/org/opensearch/agent/tools/VectorDBTool.java

```diff
@@ -28,6 +28,9 @@
 @ToolAnnotation(VectorDBTool.TYPE)
 public class VectorDBTool extends AbstractRetrieverTool {
     public static final String TYPE = "VectorDBTool";
+
+    public static String DEFAULT_DESCRIPTION =
```

Comment:
I have written description on **VectorDBTool** and **RAGTool**. Can you help me check them?

Reply:
I took the description and adapt the format from abstractRetrieverTool and other tools "Use this tool to do something"